fgOpenGL/ProviderGL.cpp

```diff
@@ -165,6 +165,9 @@ FG_Caps Provider::GetCaps(FG_GraphicsInterface* self)
   glGetIntegerv(GL_MAX_MESH_VIEWS_NV, &caps.openGL.max_mesh_views);
   GetVec3i(GL_MAX_MESH_WORK_GROUP_SIZE_NV, caps.openGL.max_mesh_work_group_size);
   GetVec3i(GL_MAX_TASK_WORK_GROUP_SIZE_NV, caps.openGL.max_task_work_group_size);
+  if(GLError __e{ "GetCaps", __FILE__, __LINE__ }; __e.has_error()) {
```

Comment:
This is equivalent to the `GL_ERROR` macro but logs instead of returning the error.